pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackend.scala

```diff
@@ -85,13 +87,35 @@ object SparkRBackend {
     try {
       // bind to random port
       val boundPort = sparkRBackend.init()
+      val serverSocket = new ServerSocket(0, 1)
+      val listenPort = serverSocket.getLocalPort()
```

Comment:
My take was that we should be only doing this when R launches JVM, and not JVM launches R (SparkRRunner case)